frontend/app_flowy/packages/appflowy_editor/lib/src/extensions/text_node_extensions.dart

```diff
@@ -6,7 +6,7 @@ import 'package:appflowy_editor/src/document/text_delta.dart';
 import 'package:appflowy_editor/src/document/built_in_attribute_keys.dart';
 
 extension TextNodeExtension on TextNode {
-  dynamic getAttributeInSelection(Selection selection, String styleKey) {
+  T? getAttributeInSelection<T>(Selection selection, String styleKey) {
```

Comment:
I suggest checking the type of the returned value equals `T`.

Reply:
Otherwise, it will raise an error if the type of the returned value is not equal to T.